Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -82,6 +82,7 @@ - (void)invalidate
   }];
 
   RCTSRWebSocket *webSocket = [[RCTSRWebSocket alloc] initWithURLRequest:request protocols:protocols];
+  [webSocket setDelegateDispatchQueue: _methodQueue];
```

Comment:
nit: remove space, please follow implied style

Reply:
✅ sorry bout that.